templates/repo/commits_table.tmpl

```diff
@@ -35,7 +35,12 @@
 								<img class="ui avatar image" src="{{AvatarLink .Author.Email}}" alt=""/>&nbsp;&nbsp;{{.Author.Name}}
 							{{end}}
 						</td>
+
+						{{if gt .ParentCount 1}}
+						<td class="message collapsing has-emoji grey text">
```

Comment:
Disagree. Duplicated majority of content is worse, unless case branches are a lot. In this case we only have two branches.